javascript/elements/updates_for_element.js

```diff
@@ -73,6 +73,35 @@ export default class UpdatesForElement extends SubscribingElement {
       }
 
       template.innerHTML = String(html[url(blocks[i])]).trim()
+
+      const reloadingTurboFrames = [
+        ...template.content.querySelectorAll(
+          'turbo-frame[src]:not([loading="lazy"])'
+        )
+      ]
+
+      await Promise.all(
+        reloadingTurboFrames.map(frame => {
```

Comment:
Is there a nightmare scenario where a lazy-loaded frame could contain additional frames that themselves might be lazy-loaded?

It seems like perhaps this should actually be a call to a recursive-capable function, if so.

Reply:
I'm definitely considering this, just need a good test bench for this